lib/jsdom/living/events/EventTarget-impl.js

```diff
@@ -34,18 +43,18 @@ class EventTargetImpl {
 
     for (let i = 0; i < this._eventListeners[type].length; ++i) {
       const listener = this._eventListeners[type][i];
-      if (listener.capture === capture && listener.callback === callback) {
+      if (listener.callback === callback) {
```

Comment:
This modification is incorrect. See step 4 here (and please add a test to make sure nobody ever removes this in the future) https://dom.spec.whatwg.org/#dom-eventtarget-addeventlistener